android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt

```diff
@@ -214,6 +214,10 @@ open class GeoWidgetFragment : Fragment(), Observer<FeatureCollection> {
       if (geometry is Point) {
         points.add(geometry)
       }
+    } ?: return
+
+    if (featureCollection.features()!!.size == 0) {
```

Comment:
Do we expect `featureCollection.features()` to never be `null`? This code has a potential of throwing `NullPointerException`.

Reply:
Nice catch! It shouldn't be possible but I'll check for that for some sanity

Reply:
Line 218 checks for the null. Do you agree?